client/my-sites/email/email-management/index.jsx

```diff
@@ -134,10 +134,17 @@ class EmailManagement extends React.Component {
 			return this.googleAppsUsersCard();
 		}
 
+		if ( config.isEnabled( 'email/titan-mvp' ) ) {
+			const chosenDomain = domainList[ 0 ];
```

Comment:
Super minor but thought I could mention it: we usually name such variable `selectedDomain`.